docs/guide.rst

```diff
@@ -22,7 +22,7 @@ or install the bleeding edge from Github:
 What is pygfx?
 --------------
 
-PyGfx is a render engine. It renders objects that are organized in a scene, and
+Pygfx is a render engine. It renders objects that are organized in a scene, and
```

Comment:
I think we used `pygfx` at the start of the sentence, because its also the name of the package. You can't import `Pygfx`

I don't have a strong opinion, but it would be nice to be consistent

Reply:
To me, starting a sentence without a capital feels weirder than capitalizing "pygfx" in these cases.